src/main/ts/core/PackageJson.ts

```diff
@@ -13,7 +13,10 @@ export interface PackageJson {
   readonly name: string;
   readonly version?: Version;
   readonly workspaces?: string[];
-  [k: string]: unknown;
+  readonly 'beehive-flow'?: {
```

Comment:
Should these be camel case, as most package.json properties normally use camel case? (eg `peerDependencies`, `devDependencies`, etc...)

Reply:
Hmmm... should I be consistent with package.json, or with the name of the tool?

Reply:
IMO it should follow normal conventions, which in this case would be camel case. I'd be curious to hear what others think though.

If we really wanted, we could keep `beehive-flow` but I think we should change `primary-workspace` to `primaryWorkspace` regardless, as it's not related to the name of the tool.

Reply:
I cameled the case.